cmd/operator/README.md

```diff
@@ -5,23 +5,41 @@ Cloud Prometheus Engine on Kubernetes.
 
 ## Run
 
-Create or update cluster resources required by the operator.
+Deploy all CRDs:
+
+``bash
+kubectl apply -f deploy/crds
+```
+
+Deploy all of the operator required configurations besides the operator
+deployment, otherwise you will have an operator deployed in addition to your
+local one.
 
 ```bash
-kubectl apply -f deploy/operator/
+kubectl apply -f deploy/operator/00-namespace.yaml
+kubectl apply -f deploy/operator/01-priority-class.yaml
+kubectl apply -f deploy/operator/02-service-account.yaml
+kubectl apply -f deploy/operator/03-role.yaml
+kubectl apply -f deploy/operator/04-rolebinding.yaml
 ```
 
-Run the operator locally (requires active kubectl context to have all permissions
-the operator needs):
+Run the operator locally (requires active kubectl context to have all
+permissions the operator needs):
```

Comment:
I think it would be better if we changed this whole readme from focusing on running the operator locally to focus on running the operator in a cluster. That really simplifies the steps, and provides a more realistic experience that is consistent with actual GMP

in that case, the steps are essentially:
1. (pre-req) set up GCP service account with [artifact registry permissions](https://cloud.google.com/kubernetes-engine/docs/troubleshooting#permission_denied_error)
2. build and push the operator with `DOCKER_PUSH=1 make operator`
3. Create CRDs with `kubectl apply-f deploy/crds`
4. Create the rest of the resources with `kubectl apply -f deploy --recursive`

Reply:
Please take a look!